### PR TITLE
Modulesync 2.12.0, Fix Travis CI pipeline etc.

### PR DESCRIPTION
**Pull request by dhoppe**
Fixes #311

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -3,4 +3,5 @@
   secure: "fH1bR1eBgA9tztMHtgdBSEOhNChC2YyoAczMzjUWgfDjinU1WOUPEprfV9upf6ZeQSTxAES36BBOrdxB6BVgdXlvLlCID+Lge1fTfi75ZX34q+eTadig/iZp3M0OveL2b5OmDv0cGbmMq8ywhgG+GPFuwPy7HF1Qjgul8UGNFOs="
   docker_sets:
     - set: centos7-64
-    - set: ubuntu1404-64
+spec/spec_helper_acceptance.rb:
+  unmanaged: true
```
Comment (ekohl) [inline .sync.yml:7]: Isn't this the default?

File: manifests/params.pp [hunk 2]
```diff
@@ -6,71 +6,49 @@
...
-            $service_provider = 'systemd'
-          } else {
-            $json_packages = [ 'rubygem-json', 'ruby-json' ]
-            $service_file_location = '/etc/init.d/jira'
-            $service_file_template = 'jira/jira.initscript.erb'
-            $service_lockfile = '/var/lock/jira'
-            $service_provider = 'debian'
-          }
-        }
-      }
```
Comment (ekohl) [inline manifests/params.pp:66]: This drops support for Debian 7 and Ubuntu 14. That's fine, but it's probably good to do that in a separate PR first so it's included in the changelog.

File: metadata.json [hunk 3]
```diff
@@ -14,15 +14,15 @@
...
-      "version_requirement": ">= 1.0.0 < 4.0.0"
+      "version_requirement": ">= 1.0.0 < 5.0.0"
     },
     {
       "name": "puppet/staging",
       "version_requirement": ">= 2.0.1 < 4.0.0"
     },
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.25.0 < 6.0.0"
```
Comment (ekohl) [inline metadata.json:25]: Should these 2 be done separate as well for the changelog?

File: spec/spec_helper_acceptance.rb [hunk 4]
```diff
@@ -1,32 +1,12 @@
...
-  hosts.each do |host|
-    if host[:platform] =~ %r{el-7-x86_64} && host[:hypervisor] =~ %r{docker}
-      on(host, "sed -i '/nodocs/d' /etc/yum.conf")
-    end
-  end
+# This file is completely managed via modulesync
+require 'voxpupuli/acceptance/spec_helper_acceptance'
+
+configure_beaker do |_host|
+  install_module_from_forge('puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact('os.family') == 'Debian'
```
Comment (ekohl) [inline spec/spec_helper_acceptance.rb:5]: I'd prefer to be explicit here
```suggestion
  install_module_from_forge_on(host, 'puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact_on(host, 'os.family') == 'Debian'
```

While we currently don't have multiple nodes, it's good to treat it as if we did

File: .sync.yml (hunk 1 above)
Comment (dhoppe) [reply .sync.yml:7]: Yes, but I wanted to make sure that this setting has not been forgotten. Otherwise someone could set it to false and break things.

File: manifests/params.pp (hunk 2 above)
Comment (dhoppe) [reply manifests/params.pp:66]: We already dropped support for Debian 7. I just picked a commit from @bastelfreak and did some cleanup:
* https://github.com/voxpupuli/puppet-jira/pull/320/commits/b216e8d77260b0aa21d6c724c94d887fc3bb2329

File: metadata.json (hunk 3 above)
Comment (dhoppe) [reply metadata.json:25]: I just picked a commit from @alexjfisher and do not think that two separate commits would make a difference:
* https://github.com/voxpupuli/puppet-jira/pull/320/commits/78052cd8bc317c018c41d61590a00dc02b5dbcad

File: spec/spec_helper_acceptance.rb (hunk 4 above)
Comment (dhoppe) [reply spec/spec_helper_acceptance.rb:5]: Thank you very much for the review / suggestions.

File: .sync.yml (hunk 1 above)
Comment (ekohl) [reply .sync.yml:7]: You'd see that on the next modulesync and it'd fail. I wouldn't copy it.

File: spec/spec_helper_acceptance.rb [hunk 5]
```diff
@@ -2,7 +2,7 @@
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
 configure_beaker do |_host|
-  install_module_from_forge('puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact('os.family') == 'Debian'
+  install_module_from_forge_on(host, 'puppetlabs-apt', '>= 4.1.0 < 8.0.0') if fact_on(host, 'os.family') == 'Debian'
```
Comment (ekohl) [inline spec/spec_helper_acceptance.rb:5]: Note I suggested the on line, but you also need to change all others + rename `_host` to `host`.

File: .sync.yml (hunk 1 above)
Comment (dhoppe) [reply .sync.yml]: Alright. I hope you are right. ;)

File: spec/spec_helper_acceptance.rb (hunk 5 above)
Comment (dhoppe) [reply spec/spec_helper_acceptance.rb:5]: Fixed it.
